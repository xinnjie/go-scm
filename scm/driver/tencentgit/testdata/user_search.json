[
  {
    "id": 176405,
    "username": "xinnjie",
    "web_url": "https://git.code.tencent.com/u/xinnjie",
    "name": "xinnjie",
    "state": "active",
    "avatar_url": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1."
  }
]