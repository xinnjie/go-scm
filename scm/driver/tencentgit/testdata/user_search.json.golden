{
  "ID": 176405,
  "Login": "xinnjie",
  "Name": "xinnjie",
  "Avatar": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1.",
  "Link": "https://git.code.tencent.com/u/xinnjie"
}